atom/browser/api/atom_api_web_contents.cc

```diff
@@ -265,13 +265,19 @@ void WebContents::DidStopLoading(content::RenderViewHost* render_view_host) {
 
 void WebContents::DidGetResourceResponseStart(
     const content::ResourceRequestDetails& details) {
+  auto context = AtomBrowserContext::Get();
```

Comment:
We should use `web_contents()->GetBrowserContext()` to get BrowserContext of current WebContents, in future we will want to get rid of global BrowserContext(#959).